pkg/service/service.go

```diff
@@ -729,6 +744,72 @@ func (s *Service) upsertService(params *lb.SVC) (bool, lb.ID, error) {
 	return new, lb.ID(svc.frontend.ID), nil
 }
 
+// upsertHealthService makes the HealthCheckNodePort available to the external IP of the service.
+func (s *Service) upsertHealthService(svc *svcInfo) error {
```

Comment:
Would this be better named `upsertNodePortHealthService`?

Reply:
Probably, I will fix it

Reply:
Fixed in https://github.com/cilium/cilium/compare/f56980fca4f8a79d20f1453ea975e825e049c111..f240423a10e417a7e1ef3259f40c9cf221586c28